watertap/unit_models/zero_order/crystallizer_zo.py

```diff
@@ -0,0 +1,92 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a crystallizer unit.
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var, Param
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_sido, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("CrystallizerZO")
+class CrystallizerZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a crystallizer unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "crystallizer"
+
+        build_sido(self)
+
+        if "tds" not in self.config.property_package.solute_set:
+            raise KeyError('TDS must be included in the solute list for determining'
+                           ' electricity intensity and power consumption of the brine concentrator unit.')
```

Comment:
```suggestion
                           ' electricity intensity and power consumption of the crystallizer unit.')
```
Is this unit any different from the `brine_concentrator` unit, @luohezhiming , @kurbansitterley ?